src/SongManager.cpp

```diff
@@ -121,6 +124,39 @@ void SongManager::Reload( bool bAllowFastLoad, LoadingWindow *ld )
 
 	// save scores before unloading songs, or the scores will be lost
 	PROFILEMAN->SaveMachineProfile();
+	GAMESTATE->SavePlayerProfiles();
+
+	std::vector<std::tuple<PlayerNumber,RString,bool>> rejoinPlayers;

```

Comment:
I would add a space before `RString` and `bool`

Reply:
fixed!